<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QMainWindow" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb500">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb100">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb50">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCoffee">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Coffee(100)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMilk">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Milk(200)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIcetea">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Icetea(300)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbChange">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
